op-plasma/damgr.go

```diff
@@ -24,15 +24,15 @@ type Input struct {
 
 func NewPlasmaDA(log log.Logger, cfg CLIConfig) *DA {
```

Comment:
We can also have a `NewPlasmaDAWithStorage` function rather than making these field external